packages/manager/apps/container/src/container/legacy/server-sidebar/universe/HostedPrivateCloudSidebar.tsx

```diff
@@ -236,6 +241,46 @@ export default function HostedPrivateCloudSidebar() {
         ],
       });
     }
+    if (feature['veeam-backup'] || feature['hycu']) {
```

Comment:
**question:** Why `veeam-backup` is checked here?

Reply:
It's because in finality we have this 2 feature in submenu. And veam-backup change is same. 

So it's to make the future rebase easier. 

I can understand if we prefer do not do that to prevent veeam-backup late